dbms/src/AggregateFunctions/AggregateFunctionNothing.h

```diff
@@ -56,6 +56,8 @@ class AggregateFunctionNothing final : public IAggregateFunctionHelper<Aggregate
 
     void insertResultInto(ConstAggregateDataPtr, IColumn & to, Arena *) const override { to.insertDefault(); }
 
+    void batchInsertSameResultInto(ConstAggregateDataPtr __restrict, IColumn &, size_t) const override {}
```

Comment:
The behavior is not the same as `insertResultInto` which actually insert a default value into to target column

Reply:
> The behavior is not the same as `insertResultInto` which actually insert a default value into to target column

changed